tools/api-markdown-documenter/src/api-item-transforms/Utilities.ts

```diff
@@ -7,44 +7,70 @@ import { DocDeclarationReference } from "@microsoft/tsdoc";
 
 import { Link } from "../Link";
 import { DocumentNode, SectionNode } from "../documentation-domain";
-import { getDocumentPathForApiItem, getLinkForApiItem } from "./ApiItemUtilities";
+import {
+	getDocumentPathForApiItem,
+	getLinkForApiItem,
+	getUnscopedPackageName,
+} from "./ApiItemUtilities";
 import { TsdocNodeTransformOptions } from "./TsdocNodeTransforms";
 import { ApiItemTransformationConfiguration } from "./configuration";
 import { wrapInSection } from "./helpers";
 
 /**
- * Helper function for creating a {@link DocumentNode} for an API item and its generated documentation contents.
+ * Helper function to generate the front matter based on the provided configuration.
+ */
+const generateFrontMatter = (
```

Comment:
Nit: We've been using function syntax elsewhere in this package. Could we use that here as well?